operations/validate.go

```diff
@@ -29,13 +29,18 @@ func Validate() cli.Command {
 		}, cli.StringSliceFlag{
 			Name:  joinFlagNames(localModulesFlagName, "lm"),
 			Usage: "specify local modules as MODULE_NAME=PATH pairs",
+		}, cli.StringFlag{
+			Name:  joinFlagNames(projectFlagName, "p"),
+			Usage: "specify project identifier",
```

Comment:
Can we make this more helpful? It may not be clear to users why they would specify a project name in order to validate a yaml file. Maybe something like "specify project identifier in order to run validation requiring project settings"